docs/05-querying-passport-attestations-onchain.md

```diff
@@ -63,4 +75,67 @@ This how the `providerVersions` is meant to be used:
   - keep the current version of the providers array, and the deprecated providers will simply not be written onchain any more (1 bit from the `providers` field of the attestation will be unused). This typically makes sense when there is only a small number of unused field
   - create a new `providers` list version. This makes sense if the number of unused bits in the `providers` field is higher and we want to reset this
 
-_[← Back to README](..#other-topics)_
\ No newline at end of file
+## How to Decode Score Attestations
+
+Score attestations are much simpler than Passport attestations. A Score attestation only contains 3 fields in its paylod:
+
+- uint256 score
+- uint32 scorer_id
+- uint8 score_decimals
+
+You can follow Steps 1 and Step 2 above to retrieve the attestation UID and score attestation.
+In order to retreive a passport attestation, there are 2 options:
+
+1. **Option A**: read from EAS and decode the attestation similar like for PAssport (see above)
+2. **Option B**: read from the 'cache' that has been implemented in the `GitcoinResolver` smart contract
+
+Given that the caching was introduced at a later point and not all scores written to EAS have been cached, we recommend the reading a users score:
+
+1. use **Option B** (fast and gas efficient)
+2. if **Option B** did not yield any result (`time` field is 0) then fallback to **Option A** as it is possible that the score you are looking for was not yet cached (it might have been issued before the cache has been introduced)
```

Comment:
Is this accurate? The contract automatically falls back, right? Also, small type above in line 89 PAssport => Passport